app/models/service.rb

```diff
@@ -1,10 +1,12 @@
+require 'ancestry'
+
 class Service < ApplicationRecord
   DEFAULT_PROCESS_DELAY_BETWEEN_GROUPS = 120
 
+  has_ancestry :orphan_strategy => :destroy
```

Comment:
@gmcculloug Can you verify the orphan_strategy here?  Seems ok to me, just want to verify that was your intent.